pkg/motor/login.go

```diff
@@ -122,33 +131,32 @@ func createWalletConfigs(id string, req mt.LoginWithKeysRequest, shards vault.Va
 	configs := make(map[party.ID]*cmp.Config)
 
 	// if a password is provided, prefer that over the DSC
-	if req.Password != "" {
-		// build recovery Config
-		recShard, err := mpc.AesDecryptWithPassword(req.Password, shards.RecoveryShard)
-		if err != nil {
-			return nil, fmt.Errorf("error decrypting recovery shard (%s): %s", recShard, err)
-		}
-
-		configs["recovery"], err = hydrateConfig(recShard)
-		if err != nil {
-			return nil, fmt.Errorf("recovery shard: %s", err)
-		}
-	} else {
-		// build DSC Config if password is not provided
-		deviceShard, ok := shards.IssuedShards[id]
-		if !ok {
-			return nil, fmt.Errorf("could not find device shard with key '%s'", id)
-		}
-		dscShard, err := mpc.AesDecryptWithKey(req.AesDscKey, deviceShard)
-		if err != nil {
-			return nil, fmt.Errorf("error decrypting DSC shard: %s", err)
-		}
+	//if req.Password != "" {
```

Comment:
Did you mean to leave this commented out? The request is not guaranteed to have a password provided so this check is important.